contents/blog/the-posthog-array-1-41-0.md

```diff
@@ -0,0 +1,170 @@
+
+> **Note:** As a result of adding persons on events and changing how queries work we now have to be more strict about merging users. Because of that, we recommend double checking your `alias` calls, and following advice above [how to upgrade a self-hosted instance to 1.41.0](/blog/the-posthog-array-1-41-0). 
+
+### New: Count of events per user
+
+![count events per user](../images/blog/array/1-41-0-count-events-per-user.png)
+
+Have you ever asked yourself '*What's the average average number of purchases per user?*' or '*What's the maximum number of forms submitted per user?'* 
+
+Questions like these used to be hard to answer with PostHog, but no more! Use the new "Count per user" aggregation mode available in Trends to analyze _how intensely_ your users use the product and its features. "Count per user" supports common statistical functions for crunching the per-user numbers: average, median, minimum, maximum, 90th/95th/99th percentile.
+
+**Bonus:** While working on this feature, we also took the opportunity to improve our aggregation selector UI: aggregation by property value is now presented with more clarity.
+
+### New: Text cards on dashboards
+
+![text cards on dashboards](../images/blog/array/1-41-0-text-card-on-dashboard.png)
+
+Previously there was no easy way to add context or links to a dashboard, meaning you may have to send long explanations when sharing a dashboard with teammates. That's why we've added the option for users on paid plans to add text cards where they can add any information they want, including metadata, images or gifs!
+
+### New: Ingestion warnings
+
+![ingestion warnings](../images/blog/array/1-41-0-ingestion.png)
+
+We've added a new page to the Data Management section which lists warnings related to data ingestion from the past 30 days. If you still try to merge identified users into others, the Ingestion Warning page is where we'll remind you that the merge got blocked.
+
+> **Note:** Self-hosted users managing kafka separately should create a new topic `clickhouse_ingestion_warnings` manually.
+
+### New: App metrics
+
+![app metrics](../images/blog/array/1-41-0-app-metrics-insight.png)
+
+Curious how well your apps are doing? Previously, you may have had to pour over the AWS logs, but now you can head to the new app metrics page to find out how many events an app has processed, how many retries were attempted and what errors may have occured. Very handy. Want to take a look? Head to the apps page in your instance and click the chart symbol for any installed app.  
+
+App metrics are only available for users on Scale or Enterprise plans. 
+
+> **Note:** Self-hosted users managing kafka separately should create a new topic `clickhouse_app_metrics` manually.
+
+### Improved: Recordings interface
```

Comment:
This is a huge deal, and we should probably have this above most of the other updates (certainly above the text in dashboards)

Reply:
lol, I already [moved it up](https://github.com/PostHog/posthog.com/pull/4512/commits/275b292ee11b335021ad20136bd20ee77d935692) by 4 places. Could be even more.